src/Microsoft.Azure.Batch.SoftwareEntitlement.Server/Startup.cs

```diff
@@ -35,6 +35,7 @@ public void ConfigureServices(IServiceCollection services)
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         [SuppressMessage("Redundancy", "RCS1163:Unused parameter.")]
+        [SuppressMessage("Redundancies in Symbol Declarations", "RECS0154:Parameter is never used")]
```

Comment:
Add `Justification="…"` to `SupressMessage`s

Reply:
Fixed.